_run/common-kind.mk

```diff
@@ -1,19 +1,41 @@
 KIND_NAME      ?= $(shell basename $$PWD)
+# NOTE: 'kind' matches the default configuration of the GH actions KinD, it's fine to use other names locally, however in GH container name is configured by engineerd/setup-kind. `kind-control-plane` is the docker image's name in GH Actions.
```

Comment:
nit for next time:

* put doc above the thing it's documenting
* wrap at 80 columns
* it's not really clear what this is talking about since `'kind'` isn't here anywhere.

Reply:
Edited, hopefully clearer now.